lib/apiml.js

```diff
@@ -100,6 +101,46 @@ ApimlConnector.prototype = {
       return this.ipAddr;
     }
   }),
+
+  checkAgent(timeout, serviceName) {
+    return new Promise((resolve, reject) => {
+      const options = {
+        method: 'GET',
```

Comment:
It looks like `host` and `port` are missing.

Reply:
You could use Eureka client's method e.g.  https://github.com/zowe/zlux-server-framework/blob/e96f3e876a0914b30684aecfabd3861bde0058e8/lib/apiml.js#L292

Reply:
i fixed it by using the values on `this` since i didnt want to spend time adjusting & testing the code on the eurekarequest function that i havent used before.